remoteappmanager/templates/page.html

```diff
@@ -52,79 +50,80 @@
       <nav class="navbar navbar-static-top" role="navigation">
         <div class="collapse navbar-collapse pull-left" id="navbar-collapse">
           {% block topbar %}
-          {% endblock %}
+          {% end %}
         </div>
 
-        <!-- Navbar Right Menu -->
-        <div class="navbar-custom-menu">
-          <ul class="nav navbar-nav">
-            <!-- User Account Menu -->
-            <li id="user-menu" class="dropdown user user-menu">
-              <!-- Menu Toggle Button -->
-              <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-                <!-- The user image in the navbar-->
-                <img src="https://www.gravatar.com/avatar/{{ user.name | gravatar_id }}" class="user-image" alt="User Image">
-                <!-- hidden-xs hides the username on small devices so only the image appears. -->
-                <span class="hidden-xs">{{ user.name }}</span>
-              </a>
-              <ul class="dropdown-menu">
-                <!-- The user image in the menu -->
-                <li class="user-header">
-                  <img src="https://www.gravatar.com/avatar/{{ user.name | gravatar_id }}" class="img-circle" alt="User Image">
-                  <p>
-                    {{ user.name }}
-                  </p>
-                </li>
-                <!-- Menu Body -->
-                <li class="user-body">
-                </li>
-                <!-- Menu Footer-->
-                <li class="user-footer">
-                  <div class="pull-right">
-                    {% block login_widget %}
-                    <a id="logout" href="{{ logout_url }}" class="btn btn-default btn-flat"><i class="fa fa-sign-out"> Sign out</i></a>
-                    {% endblock %}
-                  </div>
-                </li>
-              </ul>
-            </li>
-          </ul>
-        </div>
+        {% if user is not None %}
```

Comment:
Ok, I suspect that it's because if you login and another user connects to the server, user is not defined (you are not passing the right credentials for the process) and in that case you issue an error message. but the error message cannot be shown because it depends on a template that requires the user.

So the best solution is to have an error message with a different base template.